R/Chromatograms-peaksData.R

```diff
@@ -103,6 +103,8 @@
 NULL
 
 #' @rdname peaksData
+#' @importFrom MsCoreUtils between
+#' @export
```

Comment:
No need to use `@export` here - I think you already exported the `peaksData` method.